src/components/Button.jsx

```diff
@@ -136,22 +136,67 @@ const Core = function (
   );
 };
 
+const ButtonWithTooltip = function (
+  {
+    display = 'panel',
+    loading = false,
+    style,
+    disabled,
+    showTooltip = false,
+    tooltipText = '',
+    size,
+    ...rest
```

Comment:
Anything that is being passed through can just be part of `...rest`. I think you only need:
```
{    
  showTooltip = false,
  tooltipText = '',
  ...rest
}
```

Reply:
Removing everything else actually broke things. Curious.

Reply:
I didn't see any errors with this:
```
const ButtonWithTooltip = function (
  {
    showTooltip = false,
    tooltipText = '',
    ...rest
  },
  ref,
) {
  if (showTooltip)
    return (
      <Tooltip title={tooltipText}>
        <span>
          <CoreForwardRef ref={ref} {...rest} />
        </span>
      </Tooltip>
    );
  else
    return (
      <CoreForwardRef ref={ref} {...rest} />
    );
};
```

Reply:
🤦 Thank you. Addressed in https://github.com/WildMeOrg/codex-frontend/pull/412/commits/75e6ad4191a8ebe4465dd63000d61fa5ddf19a09